packages/environment/src/assets/base.ts

```diff
@@ -67,4 +80,43 @@ export default defineAssetList(Network.BASE, [
       rateAsset: RateAsset.USD,
     },
   },
+  {
+    decimals: 8,
+    id: "0x0555e30da8f98308edb960aa94c0db47230d2b9c",
+    name: "Wrapped BTC",
+    releases: [sulu],
+    symbol: "WBTC",
+    type: AssetType.PRIMITIVE,
+    priceFeed: {
+      type: PriceFeedType.PRIMITIVE_CHAINLINK,
+      aggregator: "0x64c911996d3c6ac71f9b455b1e8e7266bcbd848f",
```

Comment:
this is BTC/USD aggregator. I think we should use WTBC/USD  as it is available, and more in line with what we are actually pricing https://basescan.org/address/0xCCADC697c55bbB68dc5bCdf8d3CBe83CdD4E071E#readContract

Reply:
Good catch, fixed!